pyproject.toml

```diff
@@ -13,7 +13,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.8,<3.13"
```

Comment:
Hm, our current recommendation for users is to install Python 3.11 in these systems, since Python 3.12 was only recently released. The CI runners should follow this suggestion as well.

I see that macOS is stuck on 3.10, but I think another PR of yours is bumping that, right?